apps/app-article-server/src/browser/components/article-content.js

```diff
@@ -251,6 +259,25 @@ class Content extends Component {
     );
   }
 
+  handleInternalLinks(htmlString) {
+    const validHostnames = ['www.hbl.fi', 'www.ostnyland.fi', 'www.vastranyland.fi']
+
+    const $ = cheerio.load(htmlString, null, false)
+    const $links = $('a')
+    $links.each((i, el) => {
+      const $el = $(el)
+      const urlObj = new URL($el.attr('href'))
+
+      if(validHostnames.includes(urlObj.hostname) && urlObj.pathname.startsWith('/artikel/')) {
+        let linkUuid = urlObj.pathname.slice(9)
```

Comment:
And the 9 here could be replaced with "/artikel/".length or something similar, to show what it actually is removing.

Reply:
Good comment, added that.